app/views/waste_carriers_engine/renewal_complete_forms/new.html.erb

```diff
@@ -11,7 +11,11 @@
 
       <div class="govuk-box-highlight">
         <h1 class="heading-xlarge"><%= t(".heading") %></h1>
-        <p class="font-large"><%= t(".highlight_text") %><br><span class="strong"><%= @renewal_complete_form.reg_identifier %></span></p>
+        <p class="font-large">
+          <%= t(".highlight_text") %>
+          <br>
+          <span class="strong" id="reg-identifier"><%= @renewal_complete_form.reg_identifier %></span>
```

Comment:
This looks like a mismatch between `-` and `_` here

Reply:
This is just adding new lines to make the code more readable. The only change is the actual addition of the id attribute